README.md

```diff
@@ -661,6 +661,7 @@ Table of Contents
   * [pages.github.com](https://pages.github.com/) — Hosting static site directly from GitHub repository
   * [pantheon.io](https://pantheon.io/) — Drupal and WordPress hosting, automated DevOps and scalable infrastructure. Free for developers and agencies
   * [pony.icu](https://pony.icu/) — Free Web Hosting with Unlimited Disk Space, Unlimited Bandwidth and Unlimited Websites from PonyICU. PHP and MySQL included.
+  * [presslabs.com](https://www.presslabs.com/pricing/) - Free cloud-native platform based on Kubernetes to manage and scale WordPress. Developer accounts are free of charge.
```

Comment:
```Developer accounts are free of charge. If you work on a customers' site you'll get by default a set of powerful tooling to make best use of your time and be sure to deliver best results.```

Does this mean developers get a wordpress instance of their own to use OR that they can only manage their customers paid accounts using their free account?

Reply:
I'm not too sure to be honest, I have no association with the company. I just found it on a search for managed wordpress providers relevant to developers. 

I can create an account with their service and report back later what exactly it means.

Reply:
that would be great if you could, as its written there I dont think it would fit the list - but maybe they do give you a wordpress account for a blog then it would

Reply:
It seems like their website was misleading, I'm not finding anything actually *free* for developers other than the tooling. Sorry about the trouble, I'll just close this PR out.